milestone3/frontend/src/recipeData.ts

```diff
@@ -0,0 +1,27 @@
+  instructions: ["Start by cutting tomatoes into small wedges and finely chop the scallion.", "Crack 4 eggs into a bowl and season with ¼ teaspoon salt, ¼ teaspoon white pepper, ½ teaspoon sesame oil, and 1 teaspoon Shaoxing wine. Beat eggs for a minute.", "Preheat the wok over medium heat until it just starts to smoke. Then add 2 tablespoons of oil and immediately add the eggs. Scramble the eggs and remove from the wok immediately. Set aside.", "Add 1 more tablespoon oil to the wok, turn up the heat to high, and add the tomatoes and scallions. Stir-fry for 1 minute, and then add 2 teaspoons sugar, ½ teaspoon salt, and ¼ cup water (if your stove gets very hot and liquid tends to cook off very quickly in your wok, add a little more water). Add the cooked eggs.", "Mix everything together, cover the wok, and cook for 1-2 minutes, until the tomatoes are completely softened.", "Uncover, and continue to stir-fry over high heat until the sauce thickens to your liking. Serve!"]},
+    {name: "Hot Dry Noodles (Re Gan Mian)",
+    description: "Traditional Hot Dry Noodles, or Re Gan Mian (热干面), a sesame noodle dish from Wuhan, capital of Hubei Province in China.",
+    image: "https://bootcamp-milestone-1.netlify.app/images/hot-dry-noodles.jpg",
+    ingredients: ["3 tablespoons sesame paste or tahini", "2 tablespoons sesame oil (plus 1 ½ teaspoons, divided)", "1 ½ teaspoons dark soy sauce", "1 tablespoon light soy sauce", "½ teaspoon sugar", "salt (to taste)", "240 g about 8 oz. fresh alkaline noodles", "¼ cup chopped preserved mustard stems (or to taste)", "¼ cup chopped pickled long beans (or to taste)", "1 teaspoon garlic (minced)", "Chili oil (to taste)", "¼ cup cilantro (chopped)", "2 scallions (chopped)", "½ teaspoon Chinese black vinegar"],
+    instructions: ["Start by preparing the sauce. Add 3 tablespoons sesame paste to a bowl, and gradually stir in 2 tablespoons sesame oil to smooth out the paste.", "Then add 1 ½ teaspoons dark soy sauce and 1 tablespoon light soy sauce (or 4 ½ teaspoons of master sauce, if you have it). Add 1 teaspoon sugar, and stir until well combined. If the sauce is still too thick, add in a few drops of warm water or more master sauce to thin it out. Add salt to taste and set aside. You can prepare this sauce ahead of time.", "Boil a pot of water to cook the noodle as per package instructions, undercooking it by 1 minute to ensure they remain chewy and al dente. Drain. Shake off any excess liquid from the noodles, and put them in a large bowl.", "Toss the noodles with the remaining 1 ½ teaspoons sesame oil to prevent noodles from sticking together. Add in the preserved radish, pickled long beans, garlic, chili oil, cilantro, scallions, vinegar, and tahini sauce. Quickly toss the noodles together and serve hot."]}
```

Comment:
Hi Kim-Linh, 
Could you resubmit this milestone using your own recipes? (not the ones from the example site). If you are struggling to think of some you are welcome to look recipes up and summarize from there. Sorry we didn't catch this in milestone 1.

Reply:
Hello, I already completed Milestone 5, so I no longer have this file in my current code (I am now pulling the recipe data from mongodb). Can I just make a pull request for Milestone 5 for review?

Reply:
Hi, yes if you have Milestone 5 done you can submit that, just make sure that you have different recipes in your database once you submit, and that the instructions and ingredients are showing up correctly on each page as I mentioned in the other comments. I will close this now so you can make a new request.